sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/aad/configuration/properties/AadAuthenticationProperties.java

```diff
@@ -111,41 +111,41 @@ public class AadAuthenticationProperties implements InitializingBean {
     private int jwtSizeLimit = RemoteJWKSet.DEFAULT_HTTP_SIZE_LIMIT; /* bytes */
 
     /**
-     * The lifespan(duration) of the cached JWK set before it expires. The default value is `5m`.
+     * The lifespan(duration) of the cached JWK set before it expires.
      */
     private Duration jwkSetCacheLifespan = Duration.ofMinutes(5);
 
     /**
-     * The refresh time(duration) of the cached JWK set before it expires. The default value is `5m`.
+     * The refresh time(duration) of the cached JWK set before it expires.
      */
     private Duration jwkSetCacheRefreshTime = Duration.ofMinutes(5);
 
     /**
-     * The redirect uri after logout. For instance, 'http://localhost:8080/'.
+     * The redirect uri after logout. For instance, `http://localhost:8080/`.
```

Comment:
@Netyyyy seems like here is still backtick?

Reply:
> @Netyyyy seems like here is still backtick?

I guess the PR haven't been updated yet.
After discussing, these changes will be reverted, also will add quotes for descriptions which have backtick on doc but nothing on javadoc.

Reply:
Revert backtick to '', so resolved this conversation. Thanks